de-lib/src/main/java/org/iplantc/de/preferences/client/view/PreferencesView.ui.xml

```diff
@@ -21,6 +21,13 @@
              type="com.sencha.gxt.core.client.util.Margins">
         <ui:attributes top="5" right="5" bottom="0" left="5" />
     </ui:with>
+    
+    <ui:with type="com.sencha.gxt.widget.core.client.TabItemConfig" field="general">
+            <ui:attributes text="General" />
+    </ui:with>
+    <ui:with type="com.sencha.gxt.widget.core.client.TabItemConfig" field="notification">
+        <ui:attributes text="Notification" />
```

Comment:
These tab title strings should be externalized and refactored into the appearance.